packages/spotlight/src/spotlight.js

```diff
@@ -335,25 +334,6 @@ const Spotlight = (function () {
 				if (_5WayKeyHold && !isContainer5WayHoldable(currentContainerId)) {
 					return false;
 				}
-
-				if (getContainerConfig(currentContainerId).straightOnlyLeave) {
-					const currentContainerRect = getContainerRect(currentContainerId);
-					const nextContainerRect = getContainerRect(last(nextContainerIds));
-
-					// prevent focus for obliquely leaving straightOnlyLeave containers
-					if (
-						(
-							(direction === 'left' || direction === 'right') &&
-							(nextContainerRect.bottom < currentContainerRect.top || nextContainerRect.top > currentContainerRect.bottom)
-						) ||
-						(
-							(direction === 'up' || direction === 'down') &&
-							(nextContainerRect.right < currentContainerRect.left || nextContainerRect.left > currentContainerRect.right)
-						)
-					) {
-						return false;
-					}
-				}
```

Comment:
We might want to re-consolidate the multiple `if` statements above, like it was prior to the old `straightOnlyLeave` implementation
```
if (_5WayKeyHold && nextContainerIds.indexOf(currentContainerId) < 0 && !isContainer5WayHoldable(currentContainerId)) {
```

Reply:
I pushed this change :)